app/main.py

```diff
@@ -16,4 +16,9 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+mu = [my_favourite_films, marks, collection_of_coins]
+
+im = [lucky_number, pi, one_is_a_prime_number, name, profile_info]
+
+
+sorted_variables = {"mutable": mu, "immutable": im}
```

Comment:
Do not create variables `mu` and `im`, create only a `sorted_variables` dictionary and assign values in it directly.

Reply:
HOW i need to do it? I have limit of the words so I cant just do it as you said